### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: alerts/alerts.go [hunk 1]
```diff
@@ -153,7 +153,9 @@ func (instance *Alerts) Close() {
 
 	instance.ticker.Stop()
 
-	instance.storeCurrentCursor()
+	if err := instance.storeCurrentCursor(); err != nil {
+		log.Errorf("Can't store cursor %s", err)
```
Comment (al1img) [inline alerts/alerts.go:157]: add `:` before %s

File: database/database_internal_test.go [hunk 2]
```diff
@@ -754,7 +756,8 @@ func TestMultiThread(t *testing.T) {
 
 		_, err := db.GetOperationVersion()
 		if err != nil {
-			t.Fatalf("Can't get Operation Version : %s", err)
+			t.Errorf("Can't get Operation Version : %s", err)
```
Comment (al1img) [inline database/database_internal_test.go:759]: Remove space before `:`

File: monitoring/monitoring_internal_test.go [hunk 3]
```diff
@@ -387,8 +386,13 @@ func TestServices(t *testing.T) {
 		t.Fatalf("Can't stop monitoring service: %s", err)
 	}
 
-	runContainerWait("service1", cmd1)
-	runContainerWait("service2", cmd2)
+	if err = runContainerWait("service1", cmd1); err != nil {
+		t.Errorf("Can't run container %s:", err)
```
Comment (al1img) [inline monitoring/monitoring_internal_test.go:390]: Move `:` before `%s`

File: monitoring/monitoring_internal_test.go [hunk 4]
```diff
@@ -387,8 +386,13 @@ func TestServices(t *testing.T) {
...
+	}
+
+	if err = runContainerWait("service2", cmd2); err != nil {
+		t.Errorf("Can't run container %s:", err)
```
Comment (al1img) [inline monitoring/monitoring_internal_test.go:394]: ditto

File: networkmanager/netutils.go [hunk 5]
```diff
@@ -90,7 +87,15 @@ func createNetNS(name string) (err error) {
 			return aoserrors.Wrap(err)
 		}
 		defer origin.Close()
-		defer netns.Set(origin)
+		defer func() {
+			if setErr := netns.Set(origin); setErr != nil {
+				log.Errorf("Can't set current network namescape: %s", setErr)
```
Comment (al1img) [inline networkmanager/netutils.go:92]: Error log is not necessary.

File: networkmanager/networkmanager.go [hunk 6]
```diff
@@ -540,11 +544,19 @@ func (manager *NetworkManager) deleteNetwork(spID string) (err error) {
...
 
-	return nil
+	return err
 }
 
 func (manager *NetworkManager) removeServiceFromNetwork(serviceID, spID string) (err error) {
-	defer netns.DeleteNamed(serviceID)
+	defer func() {
+		if delErr := netns.DeleteNamed(serviceID); delErr != nil {
+			log.Errorf("Can't delete named network namespace: %s", delErr)
```
Comment (al1img) [inline networkmanager/networkmanager.go:553]: Error log is not necessary.

File: networkmanager/networkmanager_test.go [hunk 7]
```diff
@@ -387,7 +399,11 @@ func TestBandwidth(t *testing.T) {
 		t.Fatalf("Can't add service to network: %s", err)
 	}
 
-	go runOCIContainer(container0Path, "service0")
+	go func() {
+		if err := runOCIContainer(container0Path, "service0"); err != nil {
+			t.Errorf("Can't run container %s:", err)
```
Comment (al1img) [inline networkmanager/networkmanager_test.go:404]: Move `:` before `%s`

File: smserver/smserver_test.go [hunk 8]
```diff
@@ -336,7 +349,11 @@ func TestServiceStateProcessing(t *testing.T) {
 		t.Fatalf("Can't create: SM Server %s", err)
 	}
 
-	go smServer.Start()
+	go func() {
+		if err := smServer.Start(); err != nil {
+			t.Errorf("Can't start sm server")
```
Comment (al1img) [inline smserver/smserver_test.go:354]: add `:%s` and err

File: smserver/smserver_test.go [hunk 9]
```diff
@@ -289,7 +298,11 @@ func TestMonitoringNotifications(t *testing.T) {
...
```
Comment (al1img) [inline smserver/smserver_test.go:303]: add : `%s` and err

File: smserver/smserver_test.go [hunk 10]
```diff
@@ -178,7 +183,11 @@ func TestAlertNotifications(t *testing.T) {
...
```
Comment (al1img) [inline smserver/smserver_test.go:188]: add : `%s` and err

File: smserver/smserver_test.go [hunk 11]
```diff
@@ -117,7 +114,11 @@ func TestConnection(t *testing.T) {
 		t.Fatalf("Can't create SM server: %s", err)
 	}
 
-	go smServer.Start()
+	go func() {
+		if err := smServer.Start(); err != nil {
+			t.Errorf("Can't start sm server")
```
Comment (al1img) [inline smserver/smserver_test.go:119]: add : `%s` and err

File: servicemanager.go [hunk 12]
```diff
@@ -242,11 +239,13 @@ func newServiceManager(cfg *config.Config) (sm *serviceManager, err error) {
...
 }
 
-func (sm *serviceManager) handleChannels(ctx context.Context) (err error) {
+func (sm *serviceManager) handleChannels(ctx context.Context) {
 	for {
 		select {
 		case users := <-sm.iam.GetUsersChangedChannel():
-			sm.launcher.SetUsers(users)
+			if err := sm.launcher.SetUsers(users); err != nil {
+				log.Errorf("Can't set users %s", err)
```
Comment (al1img) [inline servicemanager.go:247]: add `:` before `%s`

File: resourcemanager/resourcemanager_internal_test.go [hunk 13]
```diff
@@ -286,8 +286,7 @@ func TestRequestDeviceResourceByName(t *testing.T) {
 	}
 
 	// request not existed device class
-	deviceResource, err = rm.RequestDeviceResourceByName("some_unavailable_device")
-	if err == nil {
+	if _, err = rm.RequestDeviceResourceByName("some_unavailable_device"); err == nil {
 		t.Fatalf("Can request resource: some_unavailable_device")
```
Comment (al1img) [inline resourcemanager/resourcemanager_internal_test.go:290]: Add `%s` and err

File: networkmanager/traffic.go [hunk 14]
```diff
@@ -276,7 +276,10 @@ func (monitor *trafficMonitoring) deleteTrafficChain(chain, rootChain string) (e
 
 	// Store traffic data to DB
 	if traffic, ok := monitor.trafficMap[chain]; ok {
-		monitor.trafficStorage.SetTrafficMonitorData(chain, traffic.lastUpdate, traffic.currentValue)
+		if err := monitor.trafficStorage.SetTrafficMonitorData(chain,
```
Comment (al1img) [inline networkmanager/traffic.go:279]: We should not return in this case: other cleanup will not be performed. Display error.

File: alerts/alerts.go (hunk 1 above)
Comment (Eabramov84) [reply alerts/alerts.go:157]: added

File: networkmanager/traffic.go (hunk 14 above)
Comment (Eabramov84) [reply networkmanager/traffic.go:279]: error added

File: alerts/alerts.go [hunk 15]
```diff
@@ -325,12 +342,16 @@ func (instance *Alerts) processJournal() (err error) {
 		}
 
 		var version uint64
+
```
Comment (al1img) [inline alerts/alerts.go:345]: Put one block `var ()` here.

File: alerts/alerts_test.go [hunk 16]
```diff
@@ -68,25 +70,18 @@ type testCursorStorage struct {
...
-}
-
 /*******************************************************************************
  * Vars
  ******************************************************************************/
 
 var systemd *dbus.Conn
 
-var errTimeout = errors.New("timeout")
+var errTimeout = aoserrors.New("timeout")
```
Comment (al1img) [inline alerts/alerts_test.go:79]: Keep errors.New here.

File: database/database.go [hunk 17]
```diff
@@ -420,21 +419,25 @@ func (db *Database) GetUsersService(users []string, serviceID string) (usersServ
 	}
 	defer rows.Close()
 
-	for rows.Next() {
-		if err = rows.Scan(&usersService.StorageFolder, &usersService.StateChecksum); err != nil {
-			return usersService, aoserrors.Wrap(err)
-		}
+	if rows.Err() != nil {
```
Comment (al1img) [inline database/database.go:422]: No need to check rows.Err here

File: database/database.go [hunk 18]
```diff
@@ -573,13 +583,13 @@ func (db *Database) GetAllOverrideEnvVars() (vars []pb.OverrideEnvVar, err error
 			}
 		}
 
-		vars = append(vars, envVar)
+		vars = append(vars, envVar) // nolint
```
Comment (al1img) [inline database/database.go:586]: which warning is here?

File: database/database.go [hunk 19]
```diff
@@ -739,28 +751,29 @@ func (db *Database) GetLayersInfo() (layersList []*pb.LayerStatus, err error) {
 	return layersList, aoserrors.Wrap(rows.Err())
 }
 
-// GetLayerInfoByDigest returns layers information by layer digest
+// GetLayerInfoByDigest returns layers information by layer digest.
 func (db *Database) GetLayerInfoByDigest(digest string) (layer pb.LayerStatus, err error) {
 	stmt, err := db.sql.Prepare("SELECT layerId, aosVersion FROM layers WHERE digest = ?")
 	if err != nil {
-		return layer, aoserrors.Wrap(err)
+		return layer, aoserrors.Wrap(err) // nolint
```
Comment (al1img) [inline database/database.go:758]: Which lint error is here?

File: iamclient/iamclient_test.go [hunk 20]
```diff
@@ -343,7 +360,7 @@ func randomString() string {
 	rand.Seed(time.Now().UnixNano())
 
 	for i := range secret {
-		secret[i] = secretSymbols[rand.Intn(len(secretSymbols))]
+		secret[i] = secretSymbols[rand.Intn(len(secretSymbols))] //nolint
```
Comment (al1img) [inline iamclient/iamclient_test.go:363]: What's wrong is here?

**Review by al1img — APPROVED** [review]

**Pull request merged**